shared/desktop/app/app-state.js

```diff
@@ -374,6 +379,8 @@ export default class AppState {
 
   _closedHandler() {
     this._clearWindow()
+    this.state.isUserActive = false
```

Comment:
should this be writing `false` after periods of inactivity, or is this flag just for if the app is running?

Reply:
This one is for when the app is closed, so that the .json file doesn't show the user as active when the app isn't even running

Reply:
ah ok! I misunderstood - I see now that the call in `renderer/index` writes to this via the inputmonitor as well.